app/src/main/java/com/kickstarter/libs/CurrentUser.java

```diff
@@ -73,7 +73,7 @@ public void refresh(final @NonNull User freshUser) {
   }
 
   @Override
-  public Observable<User> observable() {
+  public @NonNull Observable<User> observable() {
```

Comment:
our [interface's annotation of this](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/libs/CurrentUserType.java#L38) was also `@NonNull` so we should be consistent